docs/source/plugins/hook_specifications.rst

```diff
@@ -11,4 +11,25 @@ napari hook specification reference
 IO hooks
 --------
 
-.. autofunction:: napari_get_reader
\ No newline at end of file
+.. autofunction:: napari_get_reader
+.. autofunction:: napari_get_writer
+
+.. _write-single-layer-hookspecs:
+
+Single Layers IO
+''''''''''''''''
+
+The following hook specifications will be called when a user saves a single
```

Comment:
you mean like as additional hook_specifications?  I agree that the syntactic symmetry is appealing... but what behavior are you specifically imagining vs our current arrangement?  I get tripped up when I start thinking about all the things that "`path`" could actually contain... like what does `napari_read_image('my.csv')` do?  what are the responsibilities of the plugins?  what does it give us over the current `napari_get_reader`?  
It's a bit easier for the writers since we _know_ what the input to the hooks is (it's one of our layers)... but we will never really know what the `path` being read contains.  (though perhaps the point here is to assert that it contains something and fail if not?)

Reply:
I'm still open to this, but we can save for a future PR - I think if we hear from people that they struggle with the current `napari_get_reader` or if we start finding ourselves really wanting to use the `napari_read_image` then we can go down this road, but always easier to add rather than take away

Reply:
Yeah, it's fine to save this for a future PR.

> like what does napari_read_image('my.csv') do?

It searches for a plugin that can read an image from a csv, if there is one it returns that image, if there isn't, it fails. Am I missing something?

> what does it give us over the current napari_get_reader?

A simple decorator that can be added to reader functions (path -> arraylike) as-is. Basically, we can start using Dan Allan's universal reader functions as napari plugins, with ~0 work from library authors.

I guess the concern is that the fast return is now not guaranteed?